cmd/flags.go

```diff
@@ -85,6 +85,10 @@ func addIgnoreFlag(fs *pflag.FlagSet, ignoreNames *[]string) {
 	fs.StringSliceVarP(ignoreNames, "ignore", "", []string{}, "set file and directory names to ignore during loading (e.g., '.*' excludes hidden files)")
 }
 
+func addSigningAlgFlag(fs *pflag.FlagSet, alg *string, value string) {
+	fs.StringVarP(alg, "crypto-alg", "", value, "name of the signing algorithm")
```

Comment:
crypto is rather meaningless term here. how about --signature-alg?

Reply:
Renamed to `--signing-alg` to match the description.